packages/devextreme/testing/tests/DevExpress.ui.widgets/resizeHandle.markup.tests.js

```diff
@@ -67,3 +67,33 @@ QUnit.module('ResizeHandle markup', moduleConfig, () => {
         assert.strictEqual(this.$element.hasClass(VERTICAL_DIRECTION_CLASS), false);
     });
 });
+
+QUnit.module('Aria attributes', moduleConfig, () => {
+    QUnit.test('aria-label attribute should be set correctly', function(assert) {
+        this.reinit({ });
```

Comment:
why reinit with empty options?

Reply:
I don't want to rely on potential default values that might be used during component initialization inside the beforeEach hook.